Sources/Purchasing/PurchasesOrchestrator.swift

```diff
@@ -541,11 +550,15 @@ extension PurchasesOrchestrator: StoreKitWrapperDelegate {
 // MARK: Transaction state updates.
 private extension PurchasesOrchestrator {
 
-    func handlePurchasedTransaction(_ transaction: SKPaymentTransaction) {
-        receiptFetcher.receiptData(refreshPolicy: .onlyIfEmpty) { receiptData in
+    func handlePurchasedTransaction(_ transaction: StoreTransaction,
+                                    refreshPolicy: ReceiptRefreshPolicy = .onlyIfEmpty,
```

Comment:
🤔 why do we have `onlyIfEmpty` in this case? If we're calling this method, we _know_ that there was a purchase, right? I feel like we should always refresh in that case, there's a good chance that the receipt isn't up to date otherwise

Reply:
I added that default value to keep the existing behavior to the previous callers.
Looks like there is only one place: `storeKitWrapper(:updatedTransaction:)`. I suppose it makes sense for that one to use `.always` as well?

Reply:
I think so - if anything is coming in from StoreKit it should be safe to assume that there's an apple ID logged into the device

Reply:
Filed [CSDK-120]

Reply:
Done: https://github.com/RevenueCat/purchases-ios/pull/1703